src/styles/block/card.scss

```diff
@@ -0,0 +1,120 @@
+html {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: $main-text-color;
+}
+
+body {
+  padding: 32px 16px 16px;
+  width: 200px;
+  margin: 0;
+  border: 1px solid #f3f3f3;
+  border-radius: 5;
+  box-sizing: border-box;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 166px;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    margin: 0;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-bottom: 4px;
+
+    &-2 {
+      margin: 0;
+      width: 98px;
+      height: 14px;
```

Comment:
do not hardcode height - it should depend on content (you've set line-height anyway)